### ISSUE TITLE
When will the code be released?

### ISSUE THREAD

**Issue by baopin132** [lucidrains/magic3d-pytorch#4]
Happy New Year!Thanks for your implementation!When will the code be released?

**Comment by ajhamdi** [comment]
obviously, this is a very difficult paper to reproduce. I am eager to help @lucidrains ,

**Comment by yosun** [comment]
HAPPY VALENTINES DAY!

**Comment by Ma-Weijian** [comment]
Happy Easter!

**Comment by Edenzzzz** [comment]
@lucidrains I'm also eager to contribute! I have worked on Diffusion models and this paper is very exciting
